crates/runtime/src/libcalls.rs

```diff
@@ -183,6 +184,7 @@ pub mod trampolines {
     }
 }
 
+#[versioned_fn]
```

Comment:
This is one location where I think it'd be good to avoid this attribute because these are normal Rust functions whose symbols don't matter, so ideally versioning shouldn't come into play here.